test/boost/cql_query_test.cc

```diff
@@ -4313,3 +4315,198 @@ SEASTAR_TEST_CASE(test_rf_expand) {
+                    "    AND memtable_flush_period_in_ms = 0\n"
+                    "    AND min_index_interval = 128\n"
+                    "    AND read_repair_chance = 0\n"
+                    "    AND speculative_retry = '99.0PERCENTILE';\n"
+            },
+            {"CF2", "CREATE TABLE ks.\"CF2\" (\n"
+                    "    pk blob,\n"
+                    "    \"CK\" blob,\n"
+                    "    col1 blob,\n"
+                    "    col2 blob,\n"
+                    "    PRIMARY KEY (pk, \"CK\")\n"
+                    ") WITH CLUSTERING ORDER BY (\"CK\" DESC)\n"
+                    "    AND bloom_filter_fp_chance = 0.02\n"
+                    "    AND caching = {'keys': 'ALL','rows_per_partition': 'ALL'}\n"
+                    "    AND comment = ''\n"
+                    "    AND compaction = {'class': 'SizeTieredCompactionStrategy'}\n"
+                    "    AND compression = {'sstable_compression': 'org.apache.cassandra.io.compress.LZ4Compressor'}\n"
+                    "    AND crc_check_chance = 1\n"
+                    "    AND dclocal_read_repair_chance = 0.2\n"
+                    "    AND default_time_to_live = 0\n"
+                    "    AND gc_grace_seconds = 954000\n"
+                    "    AND max_index_interval = 3048\n"
+                    "    AND memtable_flush_period_in_ms = 10\n"
+                    "    AND min_index_interval = 128\n"
+                    "    AND read_repair_chance = 0\n"
+                    "    AND speculative_retry = '99.0PERCENTILE';\n"
+            },
+            {"Cf3", "CREATE TABLE ks.\"Cf3\" (\n"
+                    "    pk blob,\n"
+                    "    pk2 blob,\n"
+                    "    \"CK\" blob,\n"
+                    "    col1 blob,\n"
+                    "    col2 blob,\n"
+                    "    PRIMARY KEY ((pk, pk2), \"CK\")\n"
+                    ") WITH CLUSTERING ORDER BY (\"CK\" DESC)\n"
+                    "    AND bloom_filter_fp_chance = 0.02\n"
+                    "    AND caching = {'keys': 'ALL','rows_per_partition': 'ALL'}\n"
+                    "    AND comment = ''\n"
+                    "    AND compaction = {'class': 'SizeTieredCompactionStrategy'}\n"
+                    "    AND compression = {'sstable_compression': 'org.apache.cassandra.io.compress.LZ4Compressor'}\n"
+                    "    AND crc_check_chance = 1\n"
+                    "    AND dclocal_read_repair_chance = 0.2\n"
+                    "    AND default_time_to_live = 0\n"
+                    "    AND gc_grace_seconds = 954000\n"
+                    "    AND max_index_interval = 3048\n"
+                    "    AND memtable_flush_period_in_ms = 10\n"
+                    "    AND min_index_interval = 128\n"
+                    "    AND read_repair_chance = 0\n"
+                    "    AND speculative_retry = '99.0PERCENTILE';\n"
+            },
+            {"cf4", "CREATE TABLE ks.cf4 (\n"
```

Comment:
what does cf4 test which cf2 didn't?

Reply:
no clustering key

Reply:
Oh, I see you have a column called "CK" but it's no longer a clustering key :-) Confusing, but right. Ok.